robottelo/cli/factory.py

```diff
@@ -2264,3 +2275,315 @@ def setup_org_for_a_rh_repo(options=None):
         u'organization-id': org_id,
         u'repository-id': rhel_repo['id'],
     }
+
+
+def _extract_capsule_satellite_installer_command(text):
+    """Extract the satellite installer command from the output of
+     capsule-certs-generate command
+    """
+    cmd_start_with = 'satellite-installer'
```

Comment:
for now it support only output that is a list or tuple, any other type will be considered as a string